src/docs/product/accounts/membership.mdx

```diff
@@ -122,6 +122,14 @@ To restrict access to teams and projects, turn off "Open Membership" for the org
 
 If "Open Membership" is toggled off, team access is on an invite-only basis. Users can request to join a team and invite others to join their teams, but access requests and invitations must be approved by an Org Manager, Org Owner, or a Team Admin for that team. Org Admins who aren't part of the team also need to follow this process to join. Once they join, they automatically become a Team Admin.
 
+### Unmerge and Delete Events
+
+By default, the **Let Members Delete Events** option in **Settings > General Settings** is enabled.
+
+When this option is enabled, members are granted the `event:admin` scope and allowed to delete and unmerge events, including the delete and discard actions.
+
+If the "Let Members Delete Events" option is disabled, members will still be able to merge events, but they will not have the ability to unmerge them.
+
```

Comment:
If an org member is a team admin, will they have the ability to delete events for their projects even if the option is disabled?